src/data/polynomial/iterated_deriv.lean

```diff
@@ -0,0 +1,234 @@
+/-
+Copyright (c) 2020 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang
+-/
+
+import data.polynomial.derivative
+import logic.function.iterate
+import tactic.ring tactic.linarith
```

Comment:
```suggestion
import tactic.ring
import tactic.linarith
```
I'm not sure if it's a style/mathlib thing, I just remember #2470 changing it so that it's one import per line.